src/VideoToolbox/VTUtilities.cs

```diff
@@ -53,22 +57,26 @@ public static VTStatus ToCGImage (this CVPixelBuffer pixelBuffer, out CGImage? i
 
 #if NET
 		[SupportedOSPlatform ("macos11.0")]
-		[UnsupportedOSPlatform ("ios")]
 		[UnsupportedOSPlatform ("tvos")]
-		[UnsupportedOSPlatform ("maccatalyst")]
```

Comment:
Do we want these maccatalyst's to be removed?

Reply:
@rolfbjarne Is this what we want?

Reply:
This is fine, these two rules come into play:

* If there's no maccatalyst attribute, then look at the ios attribute (which says unsupported)
* If there are any attributes for other platforms, then it's unsupported.

Both say maccatalyst is unsupported, which is what we want, so everything is OK.